code/modules/organs/external/_external.dm

```diff
@@ -3,7 +3,7 @@
 ****************************************************/
 
 //These control the damage thresholds for the various ways of removing limbs
-#define DROPLIMB_THRESHOLD_EDGE 0.75
+#define DROPLIMB_THRESHOLD_EDGE 1.15
```

Comment:
As only melee weapons use this, I'd suggest keeping it at 1, so even with little damage they can still remove arms and the likes.

Reply:
>little damage
>remove arms
guh?? nuh